docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Use project tests
+id: use-project-tests
+description: Custom SQL query tests
+sidebar_position: 8
+tags:
+  - how-to
+  - testing
+  - sql
+---
+
+Project tests are custom data tests that check for a passing condition, such as checking that the given query doesn’t return any unexpected rows. They are based on [dbt singular tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests), but you don’t need to know dbt to use project tests in Prophecy. Project tests are useful for data users who know the important business questions but wouldn’t necessarily know how to write tests.
+
```

Comment:
I'm not sure if we want to point out so clearly that our users don't know something 😅 ... Maybe better to focus on how we make it easier?

Reply:
Done.